docs/contributing/998-archived/01-compile-targets.md

```diff
@@ -91,7 +91,7 @@ In this example, we want to sleep briefly for the Simulator target and for 30 se
 
 Compiler plugins serve to extend the capabilities of the Wing compiler. They offer customization for the compilation output, such as infrastructure definitions. For instance, a compiler plugin can be employed to enforce a specific tagging convention on all resources within the resulting infrastructure.
```

Comment:
```suggestion
Platforms extend the capabilities of the Wing compiler. They offer customization for the compilation output, such as infrastructure definitions. For instance, a platform can enforce a specific tagging convention on all resources within the resulting infrastructure.
```

Reply:
This is part of an archived doc about compilation targets, so I don't think we should update the text in it. The reason I'm still updating the link is because it is broken and we have an automatic tool that notifies us about broken links, so if I won't fix this link it will add noise.